tests/testthat/test-query-describe.R

```diff
@@ -0,0 +1,84 @@
+test_that("describe(it()) with 2 it()s can be matched", {
+  # first example for testthat::describe()
+  code_source <- glue::glue('
```

Comment:
I like how glue lets you lay this out with the correct internal indentation (internal to the specimen) and the indentation related to begin inside `test_that()` gets removed.